### ISSUE TITLE
Email: open v closed captions comment for wcag 3 draft

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#415]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0004.html) __
Hello,
I wanted to comment or vote for closed captions only making the final version of wcag 3.  Having worked in this area for 15ish years, and presenting many times on captioning requirements and best practices, unless we are dealing with some kind of artifact or old recording with burnt in captions, I cannot see any reason to allow open captions. (and even in those cases I might edit them out of the video to apply closed)  When they are blown up they become pixelated, they are not always correct, and when closed captions are applied over them (in the default position) jumbling and confusion can occur which are problematic for cognitive issues.  It's also not technologically possible to pull open captions from a video adding to time and expense.  So for guidelines which won't go live until 2022/23 at best, looking towards a more technological future and closed captions seems the better option, than looking backwards to the 1980's
Thank you,
Vance

Vance S. Martin, Ph.D.

Executive Policy Advisor for Accessibility, University of Illinois Board of Trustees Office
240A Henry Administration Building, MC-350
506 S. Wright St.
Urbana, IL 61801
217-244-0992 phone
LinkedIn Profile<http://www.linkedin.com/in/vance-martin-08855122>

**Comment by WilcoFiers** [comment]
Thank you for reviewing and commenting on the WCAG 3 first public working draft. We have an [updated working draft](https://www.w3.org/TR/wcag-3.0/) and the group is starting a new workflow process. To facilitate this transition we have recorded your concern as part of w3c/wcag3#23. You can engage in the discussion around this issue there. 

Please continue to review future drafts and provide feedback through [Github](https://github.com/w3c/wcag3).

If you disagree with closing this issue, please reopen it and add your reasons for reopening it to the comments.
